projects/opendr_ws_2/src/data_generation/data_generation/synthetic_facial_generation.py

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env python3.6
+# Copyright 2020-2022 OpenDR European Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import cv2
+import torch
+import rclpy
+from ros2_bridge.bridge import ROS2Bridge
+import numpy as np
+from sensor_msgs.msg import Image as ROS_Image
+from SyntheticDataGeneration import MultiviewDataGeneration
+import os
+import argparse
+from opendr.engine.data import Image
+from algorithm.DDFA.utils.ddfa import str2bool
+
+
+class Synthetic_Data_Generation:
+
+    def __init__(self, input_image_topic="/usb_cam/image_raw", output_image_topic="/opendr/synthetic_facial_images",
+                 device="cuda"):
+        """
+        Creates a ROS Node for SyntheticDataGeneration
+        :param input_image_topic: Topic from which we are reading the input image
+        :type input_image_topic: str
+        :param output_image_topic: Topic to which we are publishing the synthetic facial image (if None, we are not publishing
+        any image)
+        :type output_image_topic: str
```

Comment:
PEP8 line is too long.

```suggestion
        :param output_rgb_image_topic: Topic to which we are publishing the synthetic facial image (if None, no image 
        is published)
        :type output_rgb_image_topic: str
```